### ISSUE TITLE
White icons not visible in UI (3.1)

### ISSUE THREAD

**Issue by soundstorm** [inkstitch/inkstitch#3122]
![image](https://github.com/user-attachments/assets/833c326a-7933-4cb2-bd27-2f2b12394ef6)
The white icons are not visible on the white buttons, system (Windows) is set to dark mode; will check the current release also on my Mac where I currently run a dev version.

**Comment by soundstorm** [comment]
Yeah, Windows 11 and InkScape: darkmode, InkStitch: light, but with darkmode icons 🫣😂

**Comment by kaalleen** [comment]
Looks as if wxpython isn't ready for darkmode in Windows. So we can disable the dark mode buttons for now and make your issue a feature request for the future.

**Comment by rfallara** [comment]
Is there a setting or flag we can set to disable the dark mode buttons or do we have to wait for the next release?

**Comment by soundstorm** [comment]
For now: just download a dev build from the releases here.
